src/file.jl

```diff
@@ -64,24 +64,37 @@ function h5open(filename::AbstractString, mode::AbstractString = "r";
     end
 end
 
+_change_iteration_order(track_order::Bool) = if track_order
```

Comment:
This formatting is a bit weird. Just use the long form function syntax here so the indentation helps indicate where the function begins and ends.

I'm also not clear why it was necessary to move the definition. The order of method definitions should not matter.

Reply:
> Oh, I see that it was formatted like that before.

I'll change that to long def.

> Anyways, is this change actually necessary?

Yes, because `src/fileio.jl` is conditionally loaded by `Requires.jl`: https://github.com/JuliaIO/HDF5.jl/blob/ccdb3e90a5910bee954d066de7b3f775f9067bc3/src/HDF5.jl#L208.